{
  "uid" : "af9cce175916a79c",
  "name" : "User tries to pay a cart with two RPTs that exists already in GPD in invalid state",
  "fullName" : "User pays a payment carts from existing payment position via nodoInviaCarrelloRPT: User tries to pay a cart with two RPTs that exists already in GPD in invalid state",
  "historyId" : "3d26ee416cc3affe0103c3d8164ab3c5",
  "time" : {
    "start" : 1752304172253,
    "stop" : 1752304177417,
    "duration" : 5164
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
  "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
    "steps" : [ {
      "name" : "Given systems up",
      "time" : {
        "start" : 1752304172253,
        "stop" : 1752304172961,
        "duration" : 708
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Given a cart of RPTs non-multibeneficiary",
      "time" : {
        "start" : 1752304172962,
        "stop" : 1752304172962,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And a single RPT of type BBT with 1 transfers of which none are stamps",
      "time" : {
        "start" : 1752304172962,
        "stop" : 1752304172962,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And a single RPT of type BBT with 1 transfers of which none are stamps",
      "time" : {
        "start" : 1752304172962,
        "stop" : 1752304172962,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And an existing payment position related to first RPT with segregation code equals to 48 and state equals to DRAFT",
      "time" : {
        "start" : 1752304172962,
        "stop" : 1752304173333,
        "duration" : 371
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "5867ea88032a9a2d",
        "name" : "Step: [an existing payment position related to first RPT with segregation code equals to 48 and state equals to DRAFT] - Sent request",
        "source" : "5867ea88032a9a2d.txt",
        "type" : "text/plain",
        "size" : 1260
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "When the user tries to pay a cart of RPTs on EC website",
      "time" : {
        "start" : 1752304173333,
        "stop" : 1752304173846,
        "duration" : 513
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "4801a63141613284",
        "name" : "Step: [the user tries to pay a cart of RPTs on EC website] - Sent request",
        "source" : "4801a63141613284.txt",
        "type" : "text/plain",
        "size" : 15123
      }, {
        "uid" : "248fcec88600110d",
        "name" : "Step: [the user tries to pay a cart of RPTs on EC website] - Received response",
        "source" : "248fcec88600110d.txt",
        "type" : "text/plain",
        "size" : 409
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "Then conversion to new model fails in wisp-converter",
      "time" : {
        "start" : 1752304173846,
        "stop" : 1752304174880,
        "duration" : 1034
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "3f7fc68bf6b83e15",
        "name" : "Step: [conversion to new model fails in wisp-converter]: on sessionId: [a3af20f1-9117-4388-aa7a-7ef1f832c7ad] - Received response",
        "source" : "3f7fc68bf6b83e15.txt",
        "type" : "text/plain",
        "size" : 5807
      }, {
        "uid" : "fa18ff5a09a2c808",
        "name" : "Received response",
        "source" : "fa18ff5a09a2c808.attach",
        "type" : "text/html",
        "size" : 5677
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "And the KO receipt is sent",
      "time" : {
        "start" : 1752304174880,
        "stop" : 1752304177416,
        "duration" : 2536
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: There is no error matching the one searched in the field_name: operation_error_code\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"src/bdd/wisp/steps/steps.py\", line 235, in debt_position_invalid\n    steputils.check_debt_position_invalid_and_sent_ko_receipt(context)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"/home/runner/work/pagopa-platform-integration-test/pagopa-platform-integration-test/src/utility/wisp/steps_utils.py\", line 959, in check_debt_position_invalid_and_sent_ko_receipt\n    raise AssertionError('There is no error matching the one searched in the field_name: operation_error_code')\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "712b796d7dd37a3e",
        "name" : "Step: [the KO receipt is sent]: on IUV: [469153572219674] - Received response",
        "source" : "712b796d7dd37a3e.txt",
        "type" : "text/plain",
        "size" : 54933
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 8,
    "attachmentsCount" : 6,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "tag",
    "value" : "nodo_invia_carrello_rpt"
  }, {
    "name" : "tag",
    "value" : "runnable"
  }, {
    "name" : "tag",
    "value" : "unhappy_path"
  }, {
    "name" : "feature",
    "value" : "User pays a payment carts from existing payment position via nodoInviaCarrelloRPT"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "runnable", "nodo_invia_carrello_rpt", "unhappy_path" ]
  },
  "source" : "af9cce175916a79c.json",
  "parameterValues" : [ ]
}